src/data/script.ts

```diff
@@ -246,6 +255,12 @@ export const getActionType = (action: Action): ActionType => {
       ) {
         return "activate_scene";
       }
+      if (
+        (action as PlayMediaAction).service === "media_player.play_media" &&
+        !Array.isArray((action as PlayMediaAction)?.target?.entity_id)
```

Comment:
I think that we should make sure that the number of allowed keys only includes the ones we support.